types/in_op.go

```diff
@@ -6,33 +6,53 @@ import (
 )
 
 type InOp struct {
-	slice  reflect.Value
-	append AppenderFunc
+	slice reflect.Value
 }
 
 var _ ValueAppender = (*InOp)(nil)
 
-func In(slice interface{}) *InOp {
-	v := reflect.ValueOf(slice)
+func In(values ...interface{}) *InOp {
+	var v reflect.Value
+	if len(values) == 1 {
+		v = reflect.ValueOf(values[0])
+	} else {
+		v = reflect.ValueOf(values)
+	}
```

Comment:
This breaks `Query.WhereIn()` API:
```go
query := orm.Query{}
query.WhereIn("id in(?)", 1) // panic: pg.In(unsupported int)
```

Reply:
Should be fixed by https://github.com/go-pg/pg/pull/693